src/python/pants_release/generate_release_announcement.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2023 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pathlib import Path
+
+from pants_release.common import sorted_contributors
+from pants_release.git import git
+
+from pants.util.strutil import softwrap
+
+
+def get_new_contributors() -> list[str]:
+    cur_version_sha, prev_version_sha = git(
+        "log", "-2", "--pretty=format:%h", "src/python/pants/VERSION"
+    ).splitlines(keepends=False)
+    return [
+        line[3:]
+        for line in git(
+            "diff", f"{prev_version_sha}..{cur_version_sha}", "CONTRIBUTORS.md"
+        ).splitlines(keepends=False)
+        if line.startswith("++ ")
+    ]
+
+
+def announcement_text() -> str:
+    version = Path("src/python/pants/VERSION").read_text().strip()
+    cur_version_sha, prev_version_sha = git(
+        "log", "-2", "--pretty=format:%h", "src/python/pants/VERSION"
+    ).splitlines(keepends=False)
+    git_range = f"{prev_version_sha}..{cur_version_sha}"
+    all_contributors = sorted_contributors(git_range)
+    new_contributors = sorted(
```

Comment:
Was the intention to call `get_new_contributors` for this? If not, maybe that function is unused?

Reply:
Good catch, that ended up inlined, to share the git range calculation, so that function is unused and now deleted.